### DISCUSSION TITLE
Malware not detected on Security Onion as an alert

### DISCUSSION THREAD

**Discussion by ITgeevoSOC** [Security-Onion-Solutions/securityonion#7554] [Unsupported Versions] [answered]
Hi,

We tried downloading a fake malware from https://www.eicar.org/?page_id=3950 for testing purposes but we noticed that Security Onion did not detect it as an alert.

We expected to see it from the alerts as a high to critical alert but nothing was there.

Any idea why? 

Thanks.

**Comment by RagnarSTS** [comment, marked as answer]
Same reason you're not seeing credit card numbers when you go to a bank site - encrypted.
Eicar seems to have stopped providing an unencrypted download.

**Reply by ITgeevoSOC** [reply]
Hi,

Thank you for your response.

Do you recommend another fake malware for testing purposes?

**Reply by Jakes-Lab** [reply]
You might find one of the following pcaps useful for testing:
https://docs.securityonion.net/en/2.3/so-test.html#so-test
https://docs.securityonion.net/en/2.3/pcaps.html

**Reply by ITgeevoSOC** [reply]
Thank you Jakes-Lab
